code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -417,6 +417,13 @@
 		if(HAS_TRAIT(src, TRAIT_BADTOUCH))
 			to_chat(M, span_warning("[src] looks visibly upset as you pat [p_them()] on the head."))
 
+	else if ((M.zone_selected == BODY_ZONE_PRECISE_GROIN) && isfelinid(src))
```

Comment:
The tail is an organ, right? You should be checking if they have the organ rather than checking if they're a felinid

Reply:
Yes boss how do I do it?

Reply:
replace `isfelinid(src)` with `!isnull(getorgan(/whatever/the/type/path/for/tails/is))`